### PR TITLE
Jcf collection / it-academy.by / JC1-110-22 / Alena Trafimovich / Done tasks 1-2

### PR DESCRIPTION
**Pull request by AlenaTrafimovich**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [ ] approved

### CODE REVIEW

File: src/test/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    private final StringCollectionUtil stringCollectionUtil = new StringCollectionUtil();
+
+    @Test
+    public void testResetWordsByLength() {
+        List<String> list = Arrays.asList("this", "is", "lots", "of", "fun", "for", "every", "Java", "programmer");
+
+        List<String> expected = Arrays.asList("*", "is", "*", "of", "fun", "for", "every", "*", "programmer");
+
+        Assertions.assertEquals(expected, (stringCollectionUtil.resetWordsByLength(list, 4)));
+    }
```
Comment (rakovets) [inline src/test/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtilTest.java:21]: fix

File: src/test/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    @Test
+    public void testRemoveWordsByLength() {
+        Collection<String> collection = new ArrayList<>(Arrays.asList("this", "is", "lots", "of", "fun", "for", "every", "Java", "programmer"));
+
+        Collection<String> expected = Arrays.asList("is", "of", "fun", "for", "every", "programmer");
+
+        Assertions.assertEquals(expected, stringCollectionUtil.removeWordsByLength(collection, 4));
+    }
```
Comment (rakovets) [inline src/test/java/com/rakovets/course/java/core/practice/jcf_collection/StringCollectionUtilTest.java:30]: fix

**Review by rakovets — CHANGES_REQUESTED** [review]
need to fix

**Comment by AlenaTrafimovich** [comment]
I have fixed comments

**Pull request closed without merging**